### PR TITLE
feat(card): upgrade to Spectrum CSS v3.0.0

### LINKED ISSUE #279: `sp-card` doesn't fully support Spectrum CSS delivery
### Expected Behaviour
Component is implemented based on the full Spectrum CSS import.

### Actual Behaviour
The following classes aren't processed:
- `.spectrum-Card--small`
- `.spectrum-Card-quickActions`
- `.spectrum-Card-actionButton`
- `.spectrum-Card-quickActions .spectrum-Checkbox`
- `.is-focused`

The `:focus` context isn't managed correctly in the CSS/`render()` method.

### Reproduce Scenario (including but not limited to)
- visit the stories for the component in Storybook
- attempt to focus on any of the `sp-card` implementations.

### PR DESCRIPTION
**Pull request by Westbrook**
## Description
- update `sp-card` to Spectrum CSS v3.0.0
  - adds `toggles`, `selected`, `asset` attributes
    - add `change` event
  - adds `actions` and `subtitle` slots
  - adds support for `horizontal` and `small` delivery
- adds `sp-quick-actions` pattern

This should be the last blocking PR for moving `main` to Spectrum 3/[dir] support by default.

## Related Issue
fixes #279

## Motivation and Context
- keep up with Spectrum CSS v3
- expand pattern coverage
- cover card more completely

## How Has This Been Tested?
- unit tests
- expanded visual regressions

## Screenshots (if appropriate):
image
image
image
image
image
image
image
image
image

## Types of changes
- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/card/src/spectrum-config.js [hunk 1]
```diff
@@ -16,78 +16,171 @@ module.exports = {
...
+                {
+                    name: 'footer',
+                    selector: '.spectrum-Card-footer',
+                },
+                {
+                    name: 'description',
+                    selector: '.spectrum-Card-description',
                 },
             ],
+            // attributes: [
```
Comment (andrewhatran) [inline packages/card/src/spectrum-config.js:112]: Should we remove these comments?

File: packages/checkbox/test/checkbox.test.ts [hunk 2]
```diff
@@ -174,6 +174,7 @@ describe('Checkbox', () => {
         HTMLElement.prototype.focus.apply(el);
         el.focusElement.dispatchEvent(new CustomEvent('focusout'));
         await nextFrame();
+        await nextFrame();
```
Comment (andrewhatran) [inline packages/checkbox/test/checkbox.test.ts:177]: Was this checkbox test not passing before without the additional nextFrame?

Comment (Westbrook) [reply packages/checkbox/test/checkbox.test.ts:177]: It started being flaky, like maybe 1 in 20 failures... 😞

File: packages/card/src/spectrum-config.js (hunk 1 above)
Comment (Westbrook) [reply packages/card/src/spectrum-config.js:112]: Indeed!

**Comment by Westbrook** [comment]
Just caught up from `spectrum-css-3`. Needs a round trip on CI before the golden images will be up-to-date again.

**Comment by Westbrook** [comment]
@andrewhatran we're back into the Green here. Thanks for continue to follow up on this `[dir]` project with me!

**Pull request merged**